src/Pages/Home.js

```diff
@@ -1,18 +1,37 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import getOffers from '../stores/offers/offerActions';
+import Offers from '../components/Offers';
+import Loader from '../components/Loader';
 
 class Home extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
+  componentDidMount() {
+    const { fetchoffers } = this.props;
+    fetchoffers();
```

Comment:
```suggestion
    this.props.fetchoffers();
```

Reply:
Make sense of the suggestion but eslint got me this warning: 'Must use destructuring props assignment'